components/CheckDocumentation/vacuum/MxidWraparound.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const MxidWraparoundTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const warning = config.settings["warning_threshold_pct"] as number;
+  const critical = config.settings["critical_threshold_pct"] as number;
+  return (
+    <p>
+      Detects when the multixact ID utilization on the server is more than{" "}
+      <code>{warning}%</code> and creates an issue with severity "warning".
+      Escalates to "critical" once the utilization exceeds{" "}
+      <code>{critical}%</code>. Resolves once the utilization falls below the
+      threshold.
+    </p>
+  );
+};
+
+const MxidWraparoundGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { backendsUrl },
+}) => {
+  const Link = useSmartAnchor();
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        The multixact ID is close to wraparound. This is about a multixact ID,
+        not about the normal transaction ID.
+      </p>
+      <p>
+        A multixact ID is an internal identifier used support row locking by
+        multiple transactions. Multixact IDs are created when transactions use{" "}
+        <code>SELECT ... FOR UPDATE</code> to lock and update tuples.
+      </p>
+      <p>
+        When the multixact ID wraparound happens, Postgres is unable to start a
+        new multixact and essentially makes the database down. Postgres usually
```

Comment:
Wraparound doesn't really happen, though. Maybe  something like
```suggestion
        When multixact ID wraparound is very close, Postgres is unable to execute commands that require multixact IDs to be assigned
        and shuts down to avoid data corruption. Postgres usually
```